NGitLab/GetRawFileRequest.cs

```diff
@@ -0,0 +1,14 @@
+﻿namespace NGitLab;
+
+public class GetRawFileRequest
+{
+    /// <summary>
+    /// The revision of the file to get (name of the branch/tag or commit)
+    /// </summary>
+    public string Ref { get; set; } = null;
```

Comment:
- Don't initialize to default value
- When possible, declare `init` accessor rather than `set`

```suggestion
    public string Ref { get; init; }
```